blog-highlights/demystifying-appflowy-editors-codebase.md

```diff
@@ -171,132 +169,115 @@ appflowy-editor\lib\src\editor\block_component\bulleted_list_block_component\bul
 So let's outline what actually happens behind the scenes when we press a slash on the Editor and select an option, such as a Heading.
 
 1. First, when we press `\` (slash), the `SelectionMenu` appears with a bunch of `SelectionMenuItems`.
-2. When we click the Heading item a Node of the corresonding type ('heading') gets inserted in the **Editor Document Tree** using the `EditorState`.
-3. The `AppFlowyEditor` then looks at its newly added node and searches for its corresponding `BlockComponentBuilder` using the type of the Node. So it will search for a BlockComponentBuilder for type 'heading'. Once found, it executes the builder to show a Flutter representation of the Heading Node.
+1. When we click the Heading item, a Node of the corresonding type ('heading') gets inserted in the **Editor Document Tree** using the `EditorState`.
+1. The `AppFlowyEditor` then looks at its newly added node and searches for its corresponding `BlockComponentBuilder` using the type of the Node. So it will search for a BlockComponentBuilder for type 'heading'. Once found, it executes the builder to show a Flutter representation of the Heading Node.
```

Comment:
```suggestion
3. The `AppFlowyEditor` then looks at its newly added node and searches for its corresponding `BlockComponentBuilder` using the type of the Node. So it will search for a BlockComponentBuilder for type 'heading'. Once found, it executes the builder to show a Flutter representation of the Heading Node.
```